big_tests/tests/sm_SUITE.erl

```diff
@@ -1474,10 +1474,12 @@ given_fresh_user_with_spec(Spec) ->
     {User#client{jid  = JID}, Spec}.
 
 wait_for_queue_length(Pid, Length) ->
+    DebugFun = fun() -> rpc(mim(), erlang, process_info, [Pid, messages]) end,
+    Opts = #{debug_fun => DebugFun},
     mongoose_helper:wait_until(
       fun() ->
               rpc(mim(), erlang, process_info, [Pid, message_queue_len])
-      end, {message_queue_len, Length}).
+      end, {message_queue_len, Length}, Opts).
```

Comment:
Not sure I like this one, is doing two rpc calls, and they're very related. You might as well get the messages, and check if the list is empty. If it is, then condition is met, if it is not, then store that list in the error log. I'd find that a better practice, cleaner, what do you think?

Reply:
why empty, lol? The usage is 2 any messages. Good luck debuging if it is three.